src/test/librbd/journal/test_Stress.cc

```diff
@@ -70,7 +86,7 @@ TEST_F(TestJournalStress, DiscardWithPruneWriteOverlap) {
           bufferlist payload_bl;
           payload_bl.append(payload);
           auto aio_comp = new librbd::io::AioCompletion();
-          api::Io<>::aio_write(*ictx, aio_comp, 0, payload.size(),
+          api::Io<>::aio_write(*ictx, aio_comp, offset, payload.size(),
```

Comment:
@baergj I have tested this, but would like your anyway ;)

Reply:
Oh, wow, that's embarrassing for me! 😅 LGTM, thanks Ilya